README.md

```diff
@@ -262,7 +262,7 @@ The native notifications API is hosted at `nativepush-cluster1.pusher.com` and o
 
 You can send pushes by using the `notify` method, either globally or on the instance. The method takes two parameters:
 
-- `interests`: An Array of strings which represents the interests your devices are subscribed to. These are akin to channels in the DDN with less of an epehemeral nature. Note that currently, you can only send to _one_ interest.
+- `interests`: An Array of strings which represents the interests your devices are subscribed to. These are akin to channels in the DDN with less of an epehemeral nature. Note that currently, you can only publish to, at most, _ten_ interests.
```

Comment:
So as not to annoy people who are imagining their potential design while reading the docs? Pointing it out earlier is better IMO, otherwise people publishing to a dynamic set of interests may not experience the limit until they reach production...